Doc/library/netrc.rst

```diff
@@ -12,6 +12,10 @@
 
 --------------
 
+This module implements an object-oriented interface to :file:`.netrc` files.
+
+.. availability:: Unix; not supported on VxWorks, that doesn't have home directories
```

Comment:
I don't think that it's worth it to mention home directories. What do you think of this suggestion?

```suggestion
.. availability:: Unix. Not available on VxWorks.
```

Reply:
I understand that the only issue on VxWorks is that there is no concept of user directory. But how does it prevent to use the netrc module if you specify a filename?

Maybe only skip the test but revert netrc.rst changes, to leave the documentation as it is?

Reply:
Yes. VxWorks users can use netrc module by specifying a filename. Reverted this.